tests/tensorflow2/test_keras.py

```diff
@@ -694,3 +772,34 @@ def input_fn():
     assert len(tr.steps()) == 2
     assert len(tr.steps(smd.modes.TRAIN)) == 1
     assert len(tr.steps(smd.modes.EVAL)) == 1
+
+
+@pytest.mark.skip
```

Comment:
why skipping?

Reply:
Skipping it here because the github CI runs with a DLC that does not have my TF changes.